rules/windows/powershell/powershell_clear_powershell_history.yml

```diff
@@ -14,12 +14,17 @@ logsource:
     product: windows
     service: powershell
 detection:
-    keywords:
-        - 'del (Get-PSReadlineOption).HistorySavePath'
-        - 'Set-PSReadlineOption –HistorySaveStyle SaveNothing'
-        - 'Remove-Item (Get-PSReadlineOption).HistorySavePath'
-        - 'rm (Get-PSReadlineOption).HistorySavePath'
-    condition: keywords
+    command:
+        - 'del'
+        - 'Set-PSReadlineOption'
+        - 'Remove-Item'
+        - 'rm'
+    selection_1:
+        - '(Get-PSReadlineOption).HistorySavePath'
+    selection_2:
+        - '–HistorySaveStyle'
+        - 'SaveNothing'
+    condition: command and (selection_1 or selection_2)
```

Comment:
could you also please update the field naming here?
here is a good example of [powershell-based Sigma rule](https://github.com/Neo23x0/sigma/blob/master/rules/windows/powershell/powershell_invoke_obfuscation_obfuscated_iex.yml).

Reply:
Fixed logic and field missing

Reply:
modified logic a bit and updating naming according to https://github.com/Neo23x0/sigma/pull/1193#discussion_r528048002